app/browse-solutions/all.ui.test.js

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable newline-per-chained-call */

```

Comment:
can we add this to the global rule?

Reply:
it only makes sense in the context of the ui.test.js as the expects are chained. We would't want many chained called in other files so I thought disabling it in the ui.test.js files only would be the easiest way to manage it

Reply:
ok let's go with this for now.

Reply:
I have moved to the eslint file as an override just for you 💃